src/server.js

```diff
@@ -1,11 +1,9 @@
 import listen from 'socket.io';
+import events from './events';
 
```

Comment:
What are in ./events?

Reply:
In nodejs module system, this means one of them.

1. If events.js exists, this indicates ```event.js``` itself.
2. If events.js doesn't exists and events directory exists, this indicates ```event/index.js```.